R/get_comms_doc_info.R

```diff
@@ -0,0 +1,44 @@
+#' fill comms doc
+#'
+#' Take the template word documents and save them with specific information for the upcoming workshop in the corresponding Sharepoint channel
```

Comment:
```suggestion
#' Collect information for the communication document.
#' Input to this function should include: startdate, enddate, title, address, venue, starttime, endtime, slug.
#' This information is taken from te Holy Excel Sheet
```
Added information here that I propose to delete from the multiline `@param info` tag.
In addition, this function doesn't render and save the document, correct? I amended the description to only contain the information-gathering purpose of the function :)